test/java/org/opendatakit/briefcase/ui/matchers/GenericUIMatchers.java

```diff
@@ -0,0 +1,30 @@
+package org.opendatakit.briefcase.ui.matchers;
+
+import javax.swing.JDialog;
+import org.assertj.swing.fixture.DialogFixture;
+import org.assertj.swing.fixture.JFileChooserFixture;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+
+public class GenericUIMatchers {
+  public static Matcher<Object> visible() {
+    return new BaseMatcher<Object>() {
+      @Override
+      public boolean matches(Object item) {
+        if (item != null && item instanceof DialogFixture)
```

Comment:
https://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.20.2
“At run time, the result of the instanceof operator is true if the value of the RelationalExpression is not null and the reference could be cast (§15.16) to the ReferenceType without raising a ClassCastException. Otherwise the result is false.”

Reply:
So did you remove the null checks?